core/src/main/java/org/jdbi/v3/core/connection/Connections.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.core.connection;
+
+import javax.annotation.Nullable;
+import org.jdbi.v3.core.config.JdbiConfig;
+import org.jdbi.v3.meta.Beta;
+
+@Beta
+public class Connections implements JdbiConfig<Connections> {
```

Comment:
Quibble over the naming, a "query timeout" is really a property of `Statements`, not a property of `Connections`.

Reply:
Hmm, but the timeout applies to the db connection the resultset/statement keep open for lazy loading, doesn't it?